testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -45,8 +46,18 @@ class MiniAppDisplayActivity : BaseActivity() {
 
     private lateinit var viewModel: MiniAppDisplayViewModel
 
+    override fun onOptionsItemSelected(item: MenuItem): Boolean = when (item.itemId) {
+        android.R.id.home -> {
+            finish()
```

Comment:
Does using `onBackPressed()` better?

Reply:
No, using `onBackPressed()` could be conflicted with the navigation back of webview here. The purpose is closing miniapp.